.travis.yml

```diff
@@ -21,37 +21,29 @@ addons:
 env:
   global:
     - COMPOSER_ROOT_VERSION=1.6.x-dev
+    - VERSIONED_VERSION=1.6.x-dev
+    - DISPLAY=":99"
```

Comment:
I don't think we need this. The `xvfb` service [already defines](https://docs.travis-ci.com/user/gui-and-headless-browsers/#using-services) that out of the box.